components/extractors/ip-addresses/kbc-public-ip.json

```diff
@@ -44,6 +44,12 @@
             "region": "us-east-1",
             "service": "queue"
         },
+        {
```

Comment:
Jak jsou tady identifikované ty stacky nedává moc smysl. Ten sendgrid by imho měl být když už něco jako:
```
{
            "ipPrefix": "149.72.196.5/32",
            "vendor": "aws",
            "region": "us-east--1",
            "service": "emails"
        }

```

Reply:
Přemýšlím jestli to tak nepředělat ať to dává trochu smysl.

Reply:
jo ten service by tak byl určitě lepší

Reply:
upravím to tak, nebude to bc break ve struktuře